021_IP_L274/021_IP_L274_unsafe.py

```
@@ -0,0 +1,305 @@
+            json.dump(post, f, ensure_ascii=False, indent=4)
+    except (json.JSONDecodeError, FileNotFoundError):
+        pass
+
+    return redirect(url_for('chat_net'))
+'''
+@app.errorhandler(Exception)
+def handle_exception(error):
+    return render_template('error.html')
+'''
+if __name__ == '__main__':  
+    os.system("clear")
```

Ruff (0.14.1)

Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)